pkg/minikube/cluster/cluster.go

```diff
@@ -292,8 +292,10 @@ func createVirtualboxHost(config MachineConfig) drivers.Driver {
 func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	var driver interface{}
 
-	if err := config.Downloader.CacheMinikubeISOFromURL(config.MinikubeISO); err != nil {
-		return nil, errors.Wrap(err, "Error attempting to cache minikube ISO from URL")
+	if config.VMDriver != "local" {
+		if err := config.Downloader.CacheMinikubeISOFromURL(config.MinikubeISO); err != nil {
```

Comment:
the downloader is an interface, and I think it might be a little cleaner to supply a no-op implementation earlier in the code, in cmd/start where its passed into machine config.  Although this seems ok to me, I wonder if its easy enough to change it in cmd